src/headless/plugins/omemo/utils.js

```diff
@@ -710,71 +647,9 @@ async function checkOMEMOSupported (chatbox) {
     }
 }
 
-function toggleOMEMO (ev) {
-    ev.stopPropagation();
-    ev.preventDefault();
-    const toolbar_el = u.ancestor(ev.target, 'converse-chat-toolbar');
-    if (!toolbar_el.model.get('omemo_supported')) {
-        let messages;
-        if (toolbar_el.model.get('type') === _converse.CHATROOMS_TYPE) {
-            messages = [
-                __(
-                    'Cannot use end-to-end encryption in this groupchat, ' +
-                        'either the groupchat has some anonymity or not all participants support OMEMO.'
-                )
-            ];
-        } else {
-            messages = [
-                __(
-                    "Cannot use end-to-end encryption because %1$s uses a client that doesn't support OMEMO.",
-                    toolbar_el.model.contact.getDisplayName()
-                )
-            ];
-        }
-        return api.alert('error', __('Error'), messages);
-    }
-    toolbar_el.model.save({ 'omemo_active': !toolbar_el.model.get('omemo_active') });
-}
-
-export function getOMEMOToolbarButton (toolbar_el, buttons) {
-    const model = toolbar_el.model;
-    const is_muc = model.get('type') === _converse.CHATROOMS_TYPE;
-    let title;
-    if (model.get('omemo_supported')) {
-        const i18n_plaintext = __('Messages are being sent in plaintext');
-        const i18n_encrypted = __('Messages are sent encrypted');
-        title = model.get('omemo_active') ? i18n_encrypted : i18n_plaintext;
-    } else if (is_muc) {
-        title = __(
-            'This groupchat needs to be members-only and non-anonymous in ' +
-                'order to support OMEMO encrypted messages'
-        );
-    } else {
-        title = __('OMEMO encryption is not supported');
-    }
+// TODO: add headless version of toggleOMEMO function here
```

Comment:
Doesn't look to me like there's anything in `toggleOMEMO` that should go into headless.

The only "headless" code would be  `model.save({ 'omemo_active': !model.get('omemo_active') });`
and I'm not sure that necessarily requires a separate function in headless.